pyEpiabm/pyEpiabm/sweep/spatial_sweep.py

```diff
@@ -264,3 +251,39 @@ def do_infection_event(self, infector: Person, infectee: Person,
         r = random.random()
         if r < force_of_infection:
             infectee.microcell.cell.enqueue_person(infectee)
+
+    def near_neighbour(self, cell, other_cells):
```

Comment:
Would it be more logical for nearest_neighbor to be a method of the cell, rather than the sweep. Then it would take just one argument (the other_cells).

Reply:
I do agree with Richard, it would make more sense to be a method of the cell - would it be possible to move it here? Happy for it still to be called in the bind population though as you only run this once - I like that!